hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/commit/BaseCommitActionExecutor.java

```diff
@@ -235,11 +235,15 @@ protected void commit(HoodieWriteMetadata<O> result, List<HoodieWriteStat> write
       LOG.info("Committed " + instantTime);
       result.setCommitMetadata(Option.of(metadata));
       // update cols to Index as applicable
-      HoodieColumnStatsIndexUtils.updateColsToIndex(table, config, metadata,
-          (Functions.Function2<HoodieTableMetaClient, List<String>, Void>) (metaClient, columnsToIndex) -> {
-            updateColumnsToIndexForColumnStats(metaClient, columnsToIndex);
-            return null;
-          });
+      try {
+        HoodieColumnStatsIndexUtils.updateColsToIndex(table, config, metadata,
+                (Functions.Function2<HoodieTableMetaClient, List<String>, Void>) (metaClient, columnsToIndex) -> {
+                  updateColumnsToIndexForColumnStats(metaClient, columnsToIndex);
+                  return null;
+                });
+      } catch (UnsupportedOperationException uoe) {
+        LOG.warn("Failed to update col stats, bootstrap doesn't support col stats", uoe);
```

Comment:
```suggestion
        LOG.warn("Skip updating column stats index in the metadata table for bootstrap operation because it is not supported", uoe);
```